cellmlmanip/model.py

```diff
@@ -387,6 +387,20 @@ def get_derived_quantities(self):
                               and node.get('variable_type', '') not in ('state', 'free', 'parameter')]
         return sorted(derived_quantities, key=lambda var: var.order_added)
 
+    def get_display_name(self, var, ontology=None):
+        """Return a display name for the given variable.
+
+        Looks for annotation in the ontology first, then cmeta:id if present, or the name attribute if not
+        If there is an interface component, strip the name of it out of the display name
+        :param var: the variable for which to get the display name.
+        :param ontology: the namespace prefix for the ontology used.
+
+        :return: the local name from the ontology (if such an annotation exists)
+                 or the cmeta_id (if present) or the variable's name"""
+        if self.has_ontology_annotation(var, ontology):
```

Comment:
```suggestion
        if ontology is not None and self.has_ontology_annotation(var, ontology):
```

Reply:
Hhm actually it should look also for non, meaning "any old ontology"